### ISSUE TITLE
Database: Open data only.

### ISSUE THREAD

**Issue by EvaWie** [OpenEnergyPlatform/oeplatform#157]
Where do we need / want to remind the users?
I added this rule to the [DatabaseRules](http://wiki.openmod-initiative.org/wiki/DatabaseRules).
Do we want to add a reminder to the upload interface?

**Comment by Ludee** [comment]
good idea.
Perhaps use a checkbox:
[] I confirm that I only upload Open Data...

**Comment by Ludee** [comment]
good examples for code that can be adapted:

- http://www.eclipse.org/legal/CoO.php
- http://www.eclipse.org/legal/ECA.php
- http://www.eclipse.org/legal/DCO.php

Before updating data to the oedb via the OEP there must be a checkbox to confirm the **OEP Certificate of Origin**

 Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
660 York Street, Suite 102,
San Francisco, CA 94110 USA

Everyone is permitted to copy and distribute verbatim copies of this license document, **but changing it is not allowed.** 

Damn, I just changed it.

**Comment by christian-rli** [comment]
RequirementSpecificationID=12

**Comment by Dhruvin14** [comment]
I changed the copyright in Jupyter-notebook but, can you please confirm the change?
I attached the screen-shot of Jupyter-notebook. 

![Screenshot (15)](https://user-images.githubusercontent.com/52157695/60009054-76cd0b00-9675-11e9-93dd-1f5e60de9c6d.png)
![Screenshot (14)](https://user-images.githubusercontent.com/52157695/60009055-76cd0b00-9675-11e9-98fa-2131a0f6d722.png)

**Comment by Bachibouzouk** [comment]
@Dhruvin14 - I am not sure whether changing the copyright in the OEP jupyter notebook tutorials is addressing this issue. As I understood it, this copyright is linked to the institution of the tutorials' author

**Comment by christian-rli** [comment]
I am also not sure as to why this change is necessary, or related to this issue. Could you elaborate @Dhruvin14 ?

**Comment by Dhruvin14** [comment]
@Bachibouzouk 
before I understood that I need to add copyrights and licence in each tutorial where user create the database from tutorials. 
But, as we discussed, It is with uploading the metadata on the OE interface, and it seems to finish by martin. 
I will confirm the changes and let you know.

**Comment by stap-m** [comment]
This issue is confusing :confused: 
Is it already solved? Then,please close. Else, comment on what is missing @christian-rli or @Bachibouzouk. Thanks.

**Comment by christian-rli** [comment]
The issue asks for a reminder that individuals are only allowed to upload open data to the OEP. This should be done so that people are aware of what this platform is about and so that the platform won't get in trouble because people upload copyrighted material.

There should be a page on the OEP that explains what data is allowed to be uploaded (and why). [This page](https://openenergy-platform.org/legal/tou/) does most of the job, but there should be an English version of it as well. @Ludee gave a few examples of how the page could be adapted. The upload tutorials should either explain our open data policy as well, or just mention it and link to the page on the OEP.

The copyright (and license) of upload tutorials themselves is a different matter. I think those should stay unchanged.

To sum up what is missing: 

- [ ] Rework of this page with regard to the suggestions by @Ludee 
- [ ] Mention and link the page within upload tutorials / examples

**Comment by christian-rli** [comment]
Link to the document that describes [recommended and allowed licenses on the OEP](https://github.com/OpenEnergyPlatform/tutorial/blob/develop/other/tutorial_open-data-licenses.ipynb).
